frame/democracy/src/tests.rs

```diff
@@ -120,9 +120,10 @@ impl pallet_scheduler::Config for Test {
 }
 parameter_types! {
 	pub const ExistentialDeposit: u64 = 1;
+	pub const MaxLocks: u32 = 10;
 }
 impl pallet_balances::Config for Test {
-	type MaxLocks = ();
+	type MaxLocks = MaxLocks;
```

Comment:
is there such a magic trick that `()` would be a boundless vec?

Reply:
No in this case it is the opposite. `Default` for `()` is `0` so it will be a non-usable vec.